packages/integration-oauth/src/providers/auth0.ts

```diff
@@ -0,0 +1,95 @@
+import { createUrl, handleRequest, authorizationHeaders } from "../request.js";
+import { scope, provider } from "../core.js";
+
+import type { Auth } from "lucia-auth";
+import type { OAuthConfig } from "../core.js";
+
+const PROVIDER_ID = "auth0";
+
+type Config = OAuthConfig & {
+	baseUrl: string;
+	redirectUri: string;
+};
+
+export const auth0 = (auth: Auth, config: Config) => {
+	const getAuthorizationUrl = async (state: string) => {
+		const url = createUrl(`${config.baseUrl}/authorize`, {
```

Comment:
This makes sure both `https://example.com` and `https://example.com/` work
```suggestion
		const url = createUrl(new URL("/authorize", config.baseUrl).toString(), {
```

Reply:
Resolved by latest commit